love/src/redux/selectors/selectors.js

```diff
@@ -144,9 +144,30 @@ export const getM1M3HardpointMonitorData = (state) => {
   const subscriptions = ['telemetry-MTM1M3-0-hardpointMonitorData'];
   const m1m3Data = getStreamsData(state, subscriptions);
   return {
-    hardpointsBreakawayLVDT: m1m3Data['telemetry-MTM1M3-0-hardpointMonitorData']?.breakawayLVDT?.value ?? [24.5, 10.1, -11.4, 2.5, 6.7, -18.1],
-    hardpointsDisplacementLVDT: m1m3Data['telemetry-MTM1M3-0-hardpointMonitorData']?.displacementLVDT?.value ?? [153, 45, 36, 25, 98, 3],
-    hardpointsBreakawayPressure: m1m3Data['telemetry-MTM1M3-0-hardpointMonitorData']?.breakawayPressure?.value ?? [1.2, 0.3, -3.2, 0.9, 1.12, 0.75],
+    hardpointsBreakawayLVDT: m1m3Data['telemetry-MTM1M3-0-hardpointMonitorData']?.breakawayLVDT?.value ?? [
+      24.5,
+      10.1,
+      -11.4,
+      2.5,
+      6.7,
+      -18.1,
+    ],
+    hardpointsDisplacementLVDT: m1m3Data['telemetry-MTM1M3-0-hardpointMonitorData']?.displacementLVDT?.value ?? [
+      153,
+      45,
+      36,
+      25,
+      98,
+      3,
+    ],
+    hardpointsBreakawayPressure: m1m3Data['telemetry-MTM1M3-0-hardpointMonitorData']?.breakawayPressure?.value ?? [
+      1.2,
+      0.3,
+      -3.2,
+      0.9,
+      1.12,
+      0.75,
+    ],
```

Comment:
These are values set to test right? If they won't be used, they should be removed